common/lib/api-client/middleware/post.js

```diff
@@ -11,9 +11,9 @@ module.exports = function post(maxFileSize) {
       })
 
       if (req.method === 'POST' && req.data instanceof FormData) {
-        payload.req.maxContentLength = maxFileSize
-        payload.req.maxBodyLength = maxFileSize
-        payload.req.headers = {
+        req.maxContentLength = maxFileSize
```

Comment:
This doesn't have any knock-on side effects to the downloads, right?

Reply:
It's the same object, innit. e2e tests all continue to work.

Reply:
>It's the same object, innit

👍 

>e2e tests all continue to work

Not sure they would pick up if this changed though as the fixture files are small and is this not related to when things go wrong with larger files?